agent/setup.go

```diff
@@ -200,6 +200,10 @@ func getPrometheusDefs(cfg lib.TelemetryConfig, isServer bool) ([]prometheus.Gau
 			Name: []string{"raft", "leader", "oldestLogAge"},
 			Help: "This measures how old the oldest log in the leader's log store is.",
 		},
+		{
+			Name: []string{"raft", "is_leader"},
```

Comment:
ah, I had made a comment above about using a different gauge var for this maybe something like 

```
consulServerGauges := ...
```